apps/datahub/src/app/record/record-metadata/record-metadata.component.ts

```diff
@@ -70,6 +70,11 @@ export class RecordMetadataComponent {
     mergeMap((uuid) => this.sourceService.getSourceLabel(uuid))
   )
 
+  loadFeedBacks$ = this.metadataViewFacade.metadata$.pipe(
+    map((record) => record?.title),
```

Comment:
Not sure if this is a use case, but what if the record doesn't have a title but all the other fields? Is there an error somewhere that we could catch instead of relying on the presence of the title?

Reply:
Yeah, I agree that it's not the best. It's just assuming that the title is the field that has the most chances to be filled...
I did try other solutions, such as using "error$" (but there was always at least one type of error that was preventing the feedbacks from loading in a working dataset). Using "isPresent$" or "metadata$" was also useless because there is indeed a metadata (made of uniqueIdentifier and title, even the fake ones).

Reply:
Oh, ok I see. Hmm... maybe @jahow has an idea for this?

Reply:
I think you should rely on metadataViewFacade.isPresent$ instead :) see above in the HTML, other blocks are using it as well

Reply:
I did try to use it, and it didn't work, because isPresent$ is always true when the feedbacks are loading

Reply:
Oh sorry, so much for reading quickly... There might be an issue there because `isPresent$` should be precisely used for that.

Reply:
Will investigate that then :slightly_smiling_face:

Reply:
Right, well done, this must be because of this reducer: https://github.com/geonetwork/geonetwork-ui/pull/849/files#diff-4a24c2ba947113aa39efacb131d5ce9ba63caee690c409d9d321c55b30cc0b51R83-R87

I think it makes sense that way since the error clearing has to happen only once when the complete metadata starts to load.

Reply:
Okay. So is there something left to change on this PR then?